paasta_tools/chronos_tools.py

```diff
@@ -394,6 +395,15 @@ def get_healthcheck_mode(self, _):
         # Healthchecks are not supported yet in chronos
         return None
 
+    def get_desired_state_human(self):
```

Comment:
And then should the caller inject what *type* of instance it needs the output for? Should it inject the whole object? Otherwise the caller would have to "know" to inject instances or whatever.